### PR TITLE
Standardize Proto File

### PR DESCRIPTION
**Pull request by apweili**
- [x] acs's namespace modified

- [x] Split the part that does not reference acs from contract.

### CODE REVIEW

File: contract/AElf.Contracts.Election/ElectionContractReferenceState.cs [hunk 1]
```diff
@@ -16,5 +17,6 @@ public partial class ElectionContractState
         internal AEDPoSContractContainer.AEDPoSContractReferenceState AEDPoSContract { get; set; }
         internal ParliamentContractContainer.ParliamentContractReferenceState ParliamentContract { get; set; }
         internal TreasuryContractContainer.TreasuryContractReferenceState TreasuryContract { get; set; }
```
Comment (EanCuznaivy) [inline contract/AElf.Contracts.Election/ElectionContractReferenceState.cs:19]: Maybe we can remove this.

File: contract/AElf.Contracts.MultiToken/TokenContractReferenceState.cs [hunk 2]
```diff
@@ -11,5 +12,6 @@ public partial class TokenContractState
         internal ParliamentContractContainer.ParliamentContractReferenceState ParliamentContract { get; set; }
```
Comment (EanCuznaivy) [inline contract/AElf.Contracts.MultiToken/TokenContractReferenceState.cs:12]: Remove TreasuryContract.

File: contract/AElf.Contracts.CrossChain/AElf.Contracts.CrossChain.csproj [hunk 3]
```diff
@@ -29,7 +35,7 @@
             <Link>Protobuf\Proto\reference\aedpos_contract.proto</Link>
         </ContractReference>
         <ContractReference Include="..\..\protobuf\parliament_contract.proto">
-            <Link>Protobuf\Proto\reference\parliament_contract.proto</Link>
+            <Link>Protobuf\Proto\reference\parliament_contract_impl.proto</Link>
```
Comment (jason-hoopox) [inline contract/AElf.Contracts.CrossChain/AElf.Contracts.CrossChain.csproj:38]: Change to parliament_contract

File: contract/AElf.Contracts.MultiToken/TokenContract_Fees.cs [hunk 4]
```diff
@@ -594,10 +592,14 @@ private void TransferTransactionFeesToFeeReceiver(string symbol, long totalAmoun
             var transferAmount = totalAmount.Sub(burnAmount);
             if (transferAmount == 0)
                 return;
-            if (Context.GetContractAddressByName(SmartContractConstants.TreasuryContractSystemName) != null)
+            var treasuryContractName =
```
Comment (jason-hoopox) [inline contract/AElf.Contracts.MultiToken/TokenContract_Fees.cs:595]: Change to treasuryContractAddress

File: protobuf/aelf/core.proto [hunk 5]
```diff
@@ -102,3 +102,8 @@ message BinaryMerkleTree {
     Hash root = 2;
     int32 leaf_count = 3;
 }
+
+message AuthorityInfo {
```
Comment (jason-hoopox) [inline protobuf/aelf/core.proto:106]: remove

**Review by jason-hoopox — APPROVED** [review]

**Review by shiwk — APPROVED** [review]

**Pull request merged**